xbmc/Application.cpp

```diff
@@ -1073,6 +1073,7 @@ void CApplication::CreateUserDirs() const
   CDirectory::Create("special://temp/");
   CDirectory::Create("special://logpath");
   CDirectory::Create("special://temp/temp"); // temp directory for python and dllGetTempPathA
+  CDirectory::Create("special://temp/archive_cache");
```

Comment:
This will just get removed two lines further down so it's a working fix but not a proper one

Lines below should probably be something like
```
if (CDirectory::Exists(archiveCachePath)
{
  if (!CDirectory::RemoveRecursive(archiveCachePath))
    CLog::Log("thingy");
}
CDirectory::Create(archiveCachePath);